Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1077,6 +1116,57 @@ def genS(fragment,howMuch):
 
 ###end UP17
 ###
+
+### FastSim: produce sample of premixed minbias events UP18
+steps["FS_PREMIXUP18_PU50"] = merge([
+        {"cfg":"SingleNuE10_cfi",
+         "--fast":"",
+         "--conditions":"auto:phase1_2018_realistic",
+         "-s":"GEN,SIM,RECOBEFMIX,DIGI",
+         "--eventcontent":"PREMIX",
+         "--datatier":"PREMIX",
```

Comment:
datatier should be GEN-SIM-DIGI-RAW

Reply:
and more importantly, `-s` should be `GEN,SIM,RECOBEFMIX,DIGI,L1,DIGI2RAW`

Reply:
done